<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>705</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>645</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Cadastro Fornecedor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 28, 61);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item row="24" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Codigo Produto Fornecido:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Para alterar um fornecedor, digite seu CNPJ</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Estado:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>CNPJ:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Telefone:</string>
        </property>
       </widget>
      </item>
      <item row="24" column="0">
       <widget class="QPushButton" name="add">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(218, 218, 218);
border-radius:10px;</string>
        </property>
        <property name="text">
         <string>Adicionar </string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Rua:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="24" column="2" colspan="2">
       <widget class="QPushButton" name="Salvar">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(218, 218, 218);
border-radius: 10px;
</string>
        </property>
        <property name="text">
         <string>Salvar Alteração</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="Pesquisar">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(218, 218, 218);</string>
        </property>
        <property name="text">
         <string>Pesquisar</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Cadastrar um fornecedor</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Bairro:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Número:</string>
        </property>
       </widget>
      </item>
      <item row="16" column="2">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Cidade:</string>
        </property>
       </widget>
      </item>
      <item row="20" column="2">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>CEP:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_11">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="21" column="2">
       <widget class="QLineEdit" name="lineEdit_12">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="17" column="2">
       <widget class="QLineEdit" name="lineEdit_10">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="13" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>Pesquisar</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>lineEdit_10</tabstop>
  <tabstop>lineEdit_11</tabstop>
  <tabstop>lineEdit_12</tabstop>
  <tabstop>add</tabstop>
  <tabstop>Salvar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Salvar</sender>
   <signal>released()</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Salvar</sender>
   <signal>released()</signal>
   <receiver>lineEdit_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Salvar</sender>
   <signal>released()</signal>
   <receiver>lineEdit_3</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Salvar</sender>
   <signal>released()</signal>
   <receiver>lineEdit_4</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add</sender>
   <signal>released()</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add</sender>
   <signal>released()</signal>
   <receiver>lineEdit_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add</sender>
   <signal>released()</signal>
   <receiver>lineEdit_3</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add</sender>
   <signal>released()</signal>
   <receiver>lineEdit_4</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Salvar</sender>
   <signal>released()</signal>
   <receiver>lineEdit_5</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add</sender>
   <signal>released()</signal>
   <receiver>lineEdit_6</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Salvar</sender>
   <signal>released()</signal>
   <receiver>lineEdit_6</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add</sender>
   <signal>released()</signal>
   <receiver>lineEdit_7</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add</sender>
   <signal>released()</signal>
   <receiver>lineEdit_9</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add</sender>
   <signal>released()</signal>
   <receiver>lineEdit_11</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>631</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>610</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add</sender>
   <signal>released()</signal>
   <receiver>lineEdit_8</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add</sender>
   <signal>released()</signal>
   <receiver>lineEdit_10</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add</sender>
   <signal>released()</signal>
   <receiver>lineEdit_12</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>610</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Salvar</sender>
   <signal>released()</signal>
   <receiver>lineEdit_7</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>627</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Salvar</sender>
   <signal>released()</signal>
   <receiver>lineEdit_8</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Salvar</sender>
   <signal>released()</signal>
   <receiver>lineEdit_9</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Salvar</sender>
   <signal>released()</signal>
   <receiver>lineEdit_10</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Salvar</sender>
   <signal>released()</signal>
   <receiver>lineEdit_11</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Salvar</sender>
   <signal>released()</signal>
   <receiver>lineEdit_12</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
